W-15933188-ldap-feedback-truststore-kt (#2598)
mulesoft/docs-connectors: ldap/3.5/modules/ROOT/pages/ldap-connector-reference.adoc

** THROW
** FOLLOW |  A constant that consists of the name of the environment property to specify how referrals encountered by the service provider must be processed (FOLLOW, IGNORE, THROW). | IGNORE |
| Extended Configuration a| Object |  A map instance that consists of extended configuration attributes used in the context environment. When working with TLS connections, ensure that the native LDAP pooling functionality is turned off. For example, if using JNDI, do not use attributes such as `com.sun.jndi.ldap.connect.pool=true`, as it will cause problems when using TLS. |  |

> | Custom TrustStore Path a| String |  Path to the truststore that contains the certificates needed for secure authentication. Under FIPS 140-2 compliance, KeyStore and TrustStore must be in PKCS#12 format. To ensure FIPS compliance, when using a custom TrustStore path, provide your certificates in PKCS#12 format.  |  |

Reviewer: ```suggestion
| Custom TrustStore Path a| String |  Path to the truststore that contains the certificates for secure authentication. To be FIPS 140-2 compliant, the keystore, truststore, and any custom truststore paths you are using,  must be in PKCS#12 format. |  |
```

Reviewer: I know they camel cased TrustStore in the UI (don't know why). They are just truststores and keystores.
